### DISCUSSION TITLE
Search menu

### DISCUSSION THREAD

**Discussion by zubcevic** [WebGoat/WebGoat#1313] [Ideas]
I would definitely be interested in a search box for the lesson menu. So e.g. somewhere you type in XXE and the correct category opens to show the link to the lesson. 
I would think a simple element and some Javascript would be enough. Any ideas?

**Comment by aolle** [comment]
I like the idea, I understand something like an instant search, I saw that the project is using jquery and backbone for managing the collections and views, I think that you are right, with some js would be enough.

Maybe that can help:
https://stackoverflow.com/questions/18157333/backbone-js-best-practice-for-implementing-instant-search
http://jsfiddle.net/XxRD7/2/

**Reply by zubcevic** [reply]
I was fixing the issue with the dropdowns and was debugging the ui in firefox/chrome and notice a lot of remarks about deprecated stuff related to the UI. Which may become a problem in future releases of browsers.

**Reply by zubcevic** [reply]
By the way: Here is a Readme for adding another language: https://github.com/WebGoat/WebGoat/blob/develop/README_I18N.md

**Reply by zubcevic** [reply]
https://github.com/WebGoat/WebGoat/pull/1317
